recipes/smhasher/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: smhasher
+  version: "0.150.1"
+
+source:
+  fn: smhasher-0.150.1.tar.gz
+  url: https://pypi.python.org/packages/5f/b2/421c060300f1449646787273ecd7df558f6f8e4caee3d6fbe8b193cd60d4/smhasher-0.150.1.tar.gz
+  md5: 4fc61f36b141e137dd2d71ecd0167866
+
+requirements:
+  build:
+    - python
+
+  run:
+    - python
+    - libgcc
```

Comment:
@mvdbeek this is only needed if you compile something afaik, do you really need it?